src/components/SelectionList/Search/ActionCell.tsx

```diff
@@ -78,30 +88,28 @@ function ActionCell({
     }
 
     if (action === CONST.SEARCH.ACTION_TYPES.VIEW || shouldUseViewAction) {
-        const buttonInnerStyles = isSelected ? styles.buttonDefaultHovered : {};
         return isLargeScreenWidth ? (
             <Button
                 text={translate(actionTranslationsMap[CONST.SEARCH.ACTION_TYPES.VIEW])}
                 onPress={goToItem}
                 small
                 style={[styles.w100]}
-                innerStyles={buttonInnerStyles}
+                innerStyles={getButtonInnerStyles(false)}
```

Comment:
Why we're passing `shouldUseSuccessStyle` as false here ?

Reply:
It's for View action which doesn't use the success style.
https://github.com/Expensify/App/blob/890806d8cca43d1c9ae7602ccb31e4a6bc5dff4e/src/components/SelectionList/Search/ActionCell.tsx#L81